changhyun/db/migrations/20221103122853_create_users_table.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+CREATE TABLE users (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(50) NOT NULL,
+email VARCHAR(200) NOT NULL,
+profile_image VARCHAR (1000) NULL,
+password VARCHAR(200) NOT NULL,
+created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+PRIMARY KEY (id),
+CONSTRAINT users_email_ukey UNIQUE (email)
+);
+
+-- migrate:down
+DROP TABLE users;
```

Comment:
```suggestion
-- migrate:up
CREATE TABLE users (
    id INT NOT NULL AUTO_INCREMENT,
    name VARCHAR(50) NOT NULL,
    email VARCHAR(200) NOT NULL,
    profile_image VARCHAR (1000) NULL,
    password VARCHAR(200) NOT NULL,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE 
    CURRENT_TIMESTAMP,
    PRIMARY KEY (id),
    CONSTRAINT users_email_ukey UNIQUE (email)
);

-- migrate:down
DROP TABLE users;
```

가독성 확보를 위해 indentaion 수정해주세요~! 다른 마이그레이션 파일들에도 적용시켜주세요!